rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```diff
@@ -58,6 +58,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("web3");
 
+    private static final CallTransaction.Function ERROR_ABI_FUNCTION = CallTransaction.Function.fromSignature("Error", "string");
+    private static final byte[] ERROR_ABI_FUNCTION_SIGNATURE = ERROR_ABI_FUNCTION.encodeSignature(); //08c379a0
```

Comment:
Any real transaction executed in a code test (maybe using DSL), to test this signature?

Reply:
Done!